GVRf/Framework/src/org/gearvrf/scene_objects/GVRSphereSceneObject.java

```diff
@@ -204,7 +204,7 @@ private void createBody(int numStacks, int numSlices) {
                 float sinPhi2 = (float) Math.sin(phi2);
 
                 // 2-----3
-                // | |
+                // |     |
                 // 0-----1
                 float x0 = sinTheta1 * cosPhi1;
```

Comment:
Again: it would probably be better to keep the intermediates as `double` and only `(float)` at the end.